auth/authpb/auth.proto

```diff
@@ -16,7 +16,21 @@ message User {
   int64 tombstone = 3;
 }
 
+// Permission is a single entity
+message Permission {
+  bytes path = 1;
+
+  enum PermissionType {
+    READ = 0;
+    WRITE = 1;
+  }
+  int32 permissionBitmap = 2;
```

Comment:
Then is defining a new field `READWRITE` ok?

BTW, how do you think about other kind of permissions e.g. create and delete?